### PR TITLE
Enable sticky header and page-tabs

### LINKED ISSUE #119: Header & page nav should be sticky when scrolling
The page header, with reduced height, should stick to the top of the screen when scrolling.

The same applies to the in page navigation tabs, when used on a page like products or about us page.

### PR DESCRIPTION
**Pull request by jose-correia**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #119

Test URLs:
- Before: https://main--moleculardevices--hlxsites.hlx.page/
- After: https://enable-stickiness--moleculardevices--hlxsites.hlx.page/

### CODE REVIEW

File: scripts/sticky-elements.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+let LAST_SCROLL_POSITION = 0;
+
+/**
+ * Enable sticky components
+ *
+ */
+export default function enableStickyElements() {
```
Comment (mhaack) [inline scripts/sticky-elements.js:7]: I would move this to scripts.js as this is global, no need for extra HTTP request.

Comment (jose-correia) [reply scripts/sticky-elements.js:7]: Sometimes I forget this is loaded on the browser and additional files have an impact on load times 😛 
I have to start coping with gigantic blobs of code 😄

Comment (mhaack) [reply scripts/sticky-elements.js:7]: I see the method in scrips.js now this file can be removed.

**Review by mhaack — APPROVED** [review]

**Comment by paolomoz** [comment]
@jose-correia there is some space between the header and the hero image

Screenshot 2023-04-21 at 11 34 32

**Comment by mhaack** [comment]
@paolomoz that has been fixed already on the main. @jose-correia can you merge main into so we can double check

**Comment by paolomoz** [comment]
@jose-correia 2 small things:

- the original has a small transition effect (not sure exactly what it is, but for example the "request quote" orange picture is sliding to the right more gracefully
- header .mainmenu-wrapper .container is 1170px with 15px padding left and right. I think padding should be 0

**Comment by jose-correia** [comment]
@mhaack looks good!

**Review by paolomoz — APPROVED** [review]

**Pull request merged**